packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -101,6 +107,20 @@ export function makeVatKeeper(
     storage.set(`${vatID}.t.${id}`, JSON.stringify(msg));
   }
 
+  function vatStats() {
+    const objectCount = storage.get(`${vatID}.o.nextID`) - FIRST_OBJECT_ID;
```

Comment:
the storage API returns strings, so rather than relying upon JS's coercion rules here, let's use `Nat(Number(storage.get(..))) - FIRST_OBJECT_ID`.

Reply:
Done